library/init/meta/expr.lean

```diff
@@ -265,6 +265,9 @@ meta constant expr.get_delayed_abstraction_locals : expr → option (list name)
 @[inline] meta def reflected.to_expr {α : Sort u} {a : α} : reflected a → expr :=
 id
 
+/-- This is a more strongly-typed version of `expr.subst` that keeps track of the value being
+reflected. When working with `reflected`, use `` (`(λ x y, foo x y).subst ex).subst ey`` instead of
+using `` `(foo %%ex %%ey) ``. -/
```

Comment:
```suggestion
reflected. When working with `reflected`, `` `(foo %%ex %%ey) `` is syntactical sugar
for `` (`(λ x y, foo x y).subst ex).subst ey``. -/
```

Or maybe you wanted to say something different here?

Reply:
Ah!

```suggestion
reflected. To obtain a term of type `reflected _`, use `` (`(λ x y, foo x y).subst ex).subst ey`` instead of
using `` `(foo %%ex %%ey) `` (which returns an `expr`). -/
```